integration-tests/sanity/sanity_suite_test.go

```diff
@@ -41,15 +41,15 @@ var _ = BeforeSuite(func() {
 	}
 
 	// Waiting for pods to be ready
-	cl, err := kubernetes.GetClientSet()
+	clientset, err := kubernetes.GetClientSet()
 	Expect(err).NotTo(HaveOccurred())
 
-	pods, err := cl.CoreV1().Pods("openebs").List(metav1.ListOptions{})
-	Expect(err).NotTo(HaveOccurred())
-	Expect(pods).NotTo(BeNil())
-
 	status := false
 	for i := 0; i < 100; i++ {
+		pods, err := clientset.CoreV1().Pods("openebs").List(metav1.ListOptions{})
```

Comment:
Get the namespace value as "openebs" as a flag & default the flag to "openebs"

Reply:
How about making it a constant ?

Reply:
There will be cases where we want to test it against different namespaces.

Reply:
updated to take it as flag `--namespace`.